qa-tests-backend/src/test/groovy/DefaultPoliciesTest.groovy

```diff
@@ -85,7 +85,7 @@ class DefaultPoliciesTest extends BaseSpecification {
     static final private List<Deployment> DEPLOYMENTS = [
         new Deployment()
             .setName (NGINX_LATEST)
-            .setImage ("nginx")
+            .setImage ("quay.io/rhacs-eng/qa:latest") // this is docker.io/redis:7.0.8-alpine but tagged as latest
```

Comment:
why mirror redis and not nginx? there are a bunch more .groovy tests that use nginx:latest and could benefit from us defining `quay.io/rhacs-eng/qa:latest` as nginx.

Reply:
I ran into flaky tests by mirroring an _existing_ nginx version as latest. Ttbomk, it's because some tests are querying for an nginx image with the same sha as the one I tagged, and then throwing up because it got the wrong tagged one. Or it picked up the wrong deployment. Effectively the tests rely on there being only one ever. 

But let me try using a brand new nginx version and tagging that as latest only.